criu/image.c

```diff
@@ -561,6 +561,11 @@ int open_image_dir(char *dir, int mode)
 		if (img_streamer_init(dir, mode) < 0)
 			goto err;
 	} else if (opts.img_parent) {
+		if (faccessat(fd, opts.img_parent, R_OK, 0)) {
+			pr_perror("Invalid parent image directory provided\n");
```

Comment:
Nit: there is no need for `\n` here because `pr_perror` will add it to the printed string just like `perror` does